whipper/test/test_command_mblookup.py

```diff
@@ -19,12 +21,32 @@ def _mock_musicbrainz(discid, country=None, record=False):
         with open(path, "rb") as p:
             return pickle.load(p)
 
+    @staticmethod
+    def _mock_getReleaseMetadata(release_id):
+        """Mock function for whipper.common.mbngs.getReleaseMetadata."""
+        filename = 'whipper.release.{}.json'.format(release_id)
+        path = os.path.join(os.path.dirname(__file__), filename)
+        with open(path, "rb") as handle:
+            response = json.loads(handle.read().decode('utf-8'))
+        return _getMetadata(response['release'])
+
     def testMissingReleaseType(self):
         """Test that lookup for release without a type set doesn't fail."""
         # Using: Gustafsson, Österberg & Cowle - What's Up? 8 (disc 4)
         # https://musicbrainz.org/release/d8e6153a-2c47-4804-9d73-0aac1081c3b1
         mblookup.musicbrainz = self._mock_musicbrainz
         discid = "xu338_M8WukSRi0J.KTlDoflB8Y-"
         # https://musicbrainz.org/cdtoc/xu338_M8WukSRi0J.KTlDoflB8Y-
-        lookup = mblookup.MBLookup([discid], 'whipper mblookup', None)
+        lookup = mblookup.MBLookup(['-d', discid],
+                                   u'whipper mblookup', None)
+        lookup.do()
+
+    def testGetDataFromReleaseId(self):
+        """Test that lookup for a release with a specified id."""
+        # Using: The KLF - Space & Chill Out
+        # https://musicbrainz.org/release/c56ff16e-1d81-47de-926f-ba22891bd2bd
+        mblookup.getReleaseMetadata = self._mock_getReleaseMetadata
+        releaseid = 'c56ff16e-1d81-47de-926f-ba22891bd2bd'
+        lookup = mblookup.MBLookup(['-R', releaseid],
+                                   u'whipper mblookup', None)
```

Comment:
I think in Python 3 strings are Unicode by default: is that `u` prefix unneeded then?

Reply:
i have fixed this in my new commit. Thank you